week3/src/components/main/CuisineSelection.jsx

```diff
@@ -0,0 +1,39 @@
+// CuisineSelection.jsx
+import React from "react";
+import { CommonBtn, StepBtn } from "../button/Button";
+
+export default function CuisineSelection({
+  onSelectCuisine,
+  selectedCuisine,
+  goBack,
+  goForward,
+}) {
+  return (
+    <div>
+      <CommonBtn
+        onClick={() => onSelectCuisine("KOREA")}
+        selected={selectedCuisine === "KOREA"}
+      >
+        한식
+      </CommonBtn>
+      <CommonBtn
+        onClick={() => onSelectCuisine("CHINA")}
+        selected={selectedCuisine === "CHINA"}
+      >
+        중식
+      </CommonBtn>
+      <CommonBtn
+        onClick={() => onSelectCuisine("JAPAN")}
+        selected={selectedCuisine === "JAPAN"}
+      >
+        일식
+      </CommonBtn>
+      <div>
+        <StepBtn onClick={goBack}>이전으로</StepBtn>
+        <StepBtn onClick={goForward} disabled={!selectedCuisine}>
```

Comment:
이것도 요런식으로 `$`표시 붙여서 전달해주세용 !!
```suggestion
        <StepBtn onClick={goForward} $disabled={!selectedCuisine}>
```

Reply:
리팩토링할 때 바로 수정할게용 ㅎㅎ